inferred-spans/src/test/java/co/elastic/otel/profiler/asyncprofiler/JfrParserTest.java

```diff
@@ -36,9 +36,9 @@ class JfrParserTest {
 
   @Test
   void name() throws Exception {
-    // using the smallest prime number possible for the buffer
-    // should trigger most edge cases in the buffer being exhausted
-    JfrParser jfrParser = new JfrParser(ByteBuffer.allocate(113), ByteBuffer.allocate(113));
+    // Using a small buffer, but big enough to fit the largest string in the JFR file to test edge
+    // cases
+    JfrParser jfrParser = new JfrParser(ByteBuffer.allocate(368), ByteBuffer.allocate(368));
```

Comment:
[minor] where does this magic value comes from ? how to pick an appropriate value and maintain this in the future ?

Reply:
It is just a small value to trigger the edge cases in `BufferdFile`, like before.
In the new JFR-file the longest string was `368` characters, which therefore is the lower bound to not crash the parsing.